src/main/resources/static/templates/PersistentXSSInHTMLTagVulnerability/LEVEL_1/PersistentXSS.js

```diff
@@ -14,13 +17,33 @@ function addingEventListenerToLoadImageButton() {
 }
 
 function appendComment(post) {
-  postDiv.innerHTML = post.content;
```

Comment:
CWE-79: Improper Neutralization of Input During Web Page Generation ('Cross-site Scripting')
------
User controlled data in methods like `innerHTML`, `outerHTML` or `document.write` is an anti-pattern that can lead to XSS vulnerabilities